okapi-core/src/test/java/org/folio/okapi/MultiTenantTest.java

```diff
@@ -357,6 +357,20 @@ public void test1(TestContext context) {
         .get("/_/proxy/tenants/" + tenant1 + "/modules")
         .then().statusCode(403).log().ifValidationFails();
 
+    // GET /_/version without token fails
+    given()
+        .header("Content-Type", "application/json")
+        .header("X-Okapi-Tenant", tenant2)
+        .get("/_/version")
+        .then().statusCode(401).log().ifValidationFails();
+
```

Comment:
Would also fail, had you used permissionsRequired instead of permissionsRequiredTenant.

Reply:
Yes.
This test ensured that a token is required. This is different from the `/_/proxy/health` API that doesn't require a token.

Reply:
Right. I forgot that permissionsRequiredTenant is ONLY used when there's a good token involved.